### PR TITLE
Feature/tao 10346/cache container params

### PR DESCRIPTION
**Pull request by jbout**
Cache container runtime params to prevent requests to the ontology

depends on:
* https://github.com/oat-sa/generis/pull/814

required for:
* https://github.com/oat-sa/extension-tao-testqti/pull/1824

### CODE REVIEW

File: model/ContainerRuntime.php [hunk 1]
```diff
@@ -38,16 +40,21 @@ class ContainerRuntime extends LegacyRuntime
...
-    
     const PROPERTY_CONTAINER = 'http://www.tao.lu/Ontologies/TAODelivery.rdf#AssembledDeliveryContainer';
 
+    const CACHE_PREFIX = 'container:';
+
     public function getDeliveryContainer($deliveryId)
     {
-        $delivery = $this->getResource($deliveryId);
-        $containerJson = (string)$delivery->getOnePropertyValue($this->getProperty(self::PROPERTY_CONTAINER));
+        $containerJson = $this->getCache()->get(self::CACHE_PREFIX.$deliveryId);
```
[the remark this answers is not in the document]
Comment (mike-ionut-mihai-sandu-tao) [reply model/ContainerRuntime.php:49]: PSR-12 may be applied on the new code developed

[the remark this answers is not in the document]
Comment (jbout) [reply model/ContainerRuntime.php:49]: s/may/should/g done

**Comment by jbout** [comment]
While possible to provide this unit test it would require a refactoring of the DeliveryContainerRegistry

File: manifest.php [hunk 2]
```diff
@@ -30,7 +31,7 @@
   'label'       => 'Delivery Management',
   'description' => 'Manages deliveries using the ontology',
   'license'     => 'GPL-2.0',
-  'version'     => '11.8.2',
+  'version'     => '11.9.0',
     'author'      => 'Open Assessment Technologies SA',
     'requires'    => [
         'generis'     => '>=12.15.0',
```
Comment (siwane) [inline manifest.php:37]: Could you update generis to the right version pls?

Comment (jbout) [reply manifest.php:37]: was waiting for other PR to be merged as I did not know what version I would end up with

**Comment by siwane** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: manifest.php (hunk 2 above)
Comment (jbout) [reply manifest.php]: done

**Comment by mike-ionut-mihai-sandu-tao** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**